cmd/bootkube/recover.go

```diff
@@ -13,7 +13,6 @@ import (
 
 	"github.com/kubernetes-incubator/bootkube/pkg/bootkube"
 	"github.com/kubernetes-incubator/bootkube/pkg/recovery"
-	"github.com/kubernetes-incubator/bootkube/pkg/util/etcdutil"
```

Comment:
Is etcdutil used at all anymore?

Reply:
Can we remove it too?

Reply:
good catch. It is not. I'll remove the code.

Reply:
fixed in 3c946bc